build.gradle

```diff
@@ -16,11 +16,12 @@
 buildscript {
 
     ext {
-        es_version = System.getProperty("es.version", "7.6.1")
+        es_version = System.getProperty("es.version", "7.7.0")
     }
     // This isn't applying from repositories.gradle so repeating it here
     repositories {
         mavenCentral()
+        mavenLocal()
```

Comment:
We don't need this because we'll not be relying on locally built artifacts(ES, or RCA) for the distro release.